src/PROPOSAL/detail/PROPOSAL/math/Integral.cxx

```diff
@@ -746,7 +746,7 @@ double Integral::Trapezoid3S(int n, double oldSum, int stepNumber)
                         {
                             determinant = std::sqrt(determinant);
                             approX      = (bEq + determinant) / aEq;
-
+                            // Why are both following conditions equal?
```

Comment:
Maybe add a TODO so that the developer has a gerater change to find this place?
```suggestion
                            // TODO: Why are both following conditions equal?
```